client/me/billing-history/billing-history-table.jsx

```diff
@@ -58,7 +58,7 @@ class BillingHistoryTable extends React.Component {
 					href={ billingHistoryReceipt( transaction.id ) }
 					onClick={ this.handleReceiptLinkClick }
 				>
-					{ translate( 'View Receipt' ) }
+					{ translate( 'View Receipt:' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.